### ISSUE TITLE
Restyling blog

### ISSUE THREAD

**Issue by ferdi05** [meilisearch/devrel#426]

**Comment by ferdi05** [comment]
Added link to the newsletter in the header and the footer by slightly customizing the current Ghost theme

**Comment by CaroFG** [comment]
It depends on the post, in the "What's new in v0.27" the footer looks much better:
![Capture d’écran 2022-05-30 à 17 43 20](https://user-images.githubusercontent.com/48251481/171025892-ff6ab8f5-311f-4f15-a35f-1d097c19386a.png)

**Comment by ferdi05** [comment]
Yes seems so. I guess this is not super good-looking because we're using an old template that was supposed to work with v3.x of Ghost (and now it's v5.x)

**Comment by ferdi05** [comment]
We should add our own searchbar #371

**Comment by ferdi05** [comment]
We could ask Meilisearch workers what are the blog designs they love, especially technical blogs.

**Comment by ferdi05** [comment]
The theme will have to comply with SEO best practices (as described in [Ghost resources](https://ghost.org/resources/beginners-guide-to-seo/#2-pick-a-platform-that-does-the-hard-work-for-you))

**Comment by ferdi05** [comment]
Weirdly, I find the article better looking in the back office compared to their published version that I find super dismal.

**Comment by ferdi05** [comment]
We need something that helps us update blog posts and feature them without maybe having to change their date. Possibly a `Last edited on` date ?

**Comment by ferdi05** [comment]
Since the [July 2022 update](https://ghost.org/changelog/search/), we can add a searchbar. thanks @shermix for the reminder

**Comment by shermix** [comment]
@ferdi05 It's just that the native search with Ghost does not support full text search functionality yet.

**Comment by CaroFG** [comment]
Hey @meilisearch/design-team, could you check the above comment and tell us what else you would need to create a design? 🙏

**Comment by ferdi05** [comment]
The restyling will wait until we have more news on how we worked with our web agency which is currently working on the landing page and our documentation website. I don't expect this to happen before Q2 2023.
